src/index.html

```diff
@@ -17,6 +29,94 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#"
```

Comment:
The `href` attribute for the logo link is set to `#`, which is a placeholder. Consider replacing it with a valid URL or a JavaScript function to handle the click event if this is intended for navigation.